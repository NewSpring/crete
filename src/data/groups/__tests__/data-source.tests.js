import ApollosConfig from '@apollosproject/config';
import GroupDataSource from '../data-source';

ApollosConfig.loadJs({
  ROCK_MAPPINGS: {
    SERVING_GROUP_TYPE_ID: 23,
    COMMUNITY_GROUP_TYPE_ID: 25,
    FAMILY_GROUP_TYPE_ID: 10,
    FUSE_GROUP_TYPE_ID: 60,
    RALLY_GROUP_TYPE_ID: 141,
    MENTORING_GROUP_TYPE_ID: 149,
  },
});

describe('Group data sources', () => {
  let Group;
  beforeEach(() => {
    Group = new GroupDataSource();
    Group.context = { dataSources: {} };
  });
  it('gets a map of group type IDs', () => {
    expect(Group.getGroupTypeIds()).toMatchSnapshot();
  });
});
