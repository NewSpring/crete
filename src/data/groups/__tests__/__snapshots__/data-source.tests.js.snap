// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Group data sources gets a map of group type IDs 1`] = `
Array [
  23,
  25,
  10,
  60,
  141,
  149,
  156,
  157,
]
`;
